### ISSUE TITLE
The addon created almost 500000 measurements in influxdb

### ISSUE THREAD

**Issue by thesebastianf** [mrk-its/homeassistant-blitzortung#138]
any lightning strike was persisted as individual measurement in influxdb addon and killed the influxdb by creating 500k and bloating the RAM beyond any limits

**Comment by thesebastianf** [comment]
![image](https://github.com/user-attachments/assets/2101d111-7c40-408a-a011-d6a6ece3d681)

actually over 500k

**Comment by bieniu** [comment]
Already fixed here https://github.com/mrk-its/homeassistant-blitzortung/pull/135

What are your radius settings? Are you tracking lightning over a large area?

**Comment by bieniu** [comment]
You can update to version 1.3.8
https://github.com/mrk-its/homeassistant-blitzortung/releases/tag/v1.3.8

**Comment by thesebastianf** [comment]
did
influx seems unsolvable however
DROP MEASUREMENT takes like 30 sec per Measurement ... times 527.000 ... :-S

**Comment by ypid** [comment]
> Already fixed here #135 

The problem is still present on the latest master (c3cc593f7e52a24ffa6a5fd08279b70b56493f63). The core issue is that geo_location entity IDs are random as thus high cardinality. #135 does not change anything because the [Home Assistant InfluxDB integration](https://www.home-assistant.io/integrations/influxdb/) is not even capable writing the `unique_id` to InfluxDB (at least not without [my patch](https://github.com/ypid/home-assistant-core/pull/1)).

https://github.com/mrk-its/homeassistant-blitzortung/blob/c3cc593f7e52a24ffa6a5fd08279b70b56493f63/custom_components/blitzortung/geo_location.py#L163-L164

I think the fix could look like creating entities like

`geo_location.lightning_strike_<Name of the integration instance>_<index>` for for example:
`geo_location.lightning_strike_home_1`

So basically a [Circular buffer](https://en.wikipedia.org/wiki/Circular_buffer). The integration already keeps track of how many geo_location entities it creates because it deletes entities to prevent the count to go over "Max number of tracked lightnings".  I tried to propose a code change but this is not as obvious to do it in 5 minutes for me without knowing the code.

I don‘t see much value in writing lightning strikes to my InfluxDB so I solved the problem for me with this config snipit for the InfluxDB integration:

```yaml
influxdb:
  exclude:
    entity_globs:
      # Blitzortung.org lightning detector:
      - 'geo_location.lightning_strike_*'
```
